README.md

```diff
@@ -182,3 +182,20 @@ result in build failure.
 All files in the repository must contain a license header in the format mentioned in [License Header](license.header).
 
 The static code analysis framework will also validate if the license exists in the specified format.
+
+### Xpanse Security Issues
+
+The xpanse project involves the use of some sensitive information, such as AK/SK sensitive fields in
+credential management, Sensitive variable information during service deployment, etc.
+
+If there is no AES encrypted file aes_sec in the root path of the project, the above sensitive
+information will not be encrypted and decrypted using AES encryption
+
+If you need to encrypt your sensitive information, please generate an AES file (file name: aes_sec)
+under the project root path. The content of the AES key file is the encryption key 256 value
```

Comment:
For local development, the AES private key file must exist in the project root path.
For environments where the application jar is directly executed, --> mention where the private key must be located.